src/data/common/Dhis2DataElement.ts

```diff
@@ -48,16 +48,20 @@ const dataElementFields = {
             name: true,
             categoryOptions: {
                 id: true,
+                code: true,
                 name: true,
                 shortName: true,
-                formName: true,
+                displayFormName: true,
```

Comment:
I tried to remove some fields here and still type-checked but there was a runtime error. That should not happened, so I searched for `as` castings. We have this `categoryOptionCombos: getCocOrdered(dataElement.categoryCombo as D2CategoryCombo, config)`. We should never do this kind of casting. It must be removed and the function defined as `function getCocOrdered(categoryCombo: D2DataElement["categoryCombo"]`. Now we can safely remove field; if unused, no TS error should appear.